src/test/java/org/prebid/server/bidder/bizzclick/BizzclickBidderTest.java

```diff
@@ -0,0 +1,401 @@
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(identity()), null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).containsExactly(BidderError.badServerResponse("Empty SeatBid array"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfBidResponseSeatBidIsEmpty() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(response -> response
+                .seatbid(emptyList())), null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).containsExactly(BidderError.badServerResponse("Empty SeatBid array"));
+    }
+
+    @Test
+    public void makeBidsShouldRelyOnlyOnFirstSeatBid() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(
+                givenBidResponse(response -> response.seatbid(List.of(
+                        SeatBid.builder().bid(singletonList(Bid.builder().id("1").build())).build(),
+                        SeatBid.builder().bid(singletonList(Bid.builder().id("2").build())).build()))),
+                givenBidRequest(givenImp(identity())));
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).extracting(BidderBid::getBid)
+                .extracting(Bid::getId)
+                .containsExactly("1");
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyListIfBidResponseFirstSeatBidIsNull() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(response -> response
+                .seatbid(singletonList(null))), null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnEmptyListIfBidResponseFirstSeatBidBidIsNull() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(response -> response
+                .seatbid(singletonList(SeatBid.builder().build()))), null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnBidsWithCorrectMediaType() throws JsonProcessingException {
```

Comment:
I believe this should be 3 separate tests. For now it checks too much. Please, break down to simpler cases.